pkg/service/service.go

```diff
@@ -219,6 +224,84 @@ func ListWithDetailedStatus(client *occlient.Client, applicationName string) (Se
 	}, nil
 }
 
+// ListOperatorServices lists all operator backed services
+func ListOperatorServices(client *kclient.Client) error {
+	glog.V(4).Info("Getting list of services")
+
+	// First let's get the list of all the operators in the namespace
+	csvs, err := client.GetClusterServiceVersionList()
+	if err != nil {
+		return errors.Wrap(err, "Unable to list operator backed services")
+	}
+
+	var allCRInstances []unstructured.Unstructured
+
+	// let's get the Services a.k.a Custom Resources (CR) defined by each operator, one by one
+	for _, csv := range csvs.Items {
+		glog.V(4).Infof("Getting services started from operator: %s\n", csv.Name)
+		customResources := client.GetCustomResourcesFromCSV(csv)
+
+		// list and write active instances of each service/CR
+		instances, err := getCRInstances(client, customResources)
+		if err != nil {
+			return err
+		}
+
+		// assuming there are more than one instances of a CR
+		allCRInstances = append(allCRInstances, instances...)
+	}
+
+	displayServices(allCRInstances)
+
+	return nil
+}
+
+func getGVRFromCR(cr olm.CRDDescription) (group, version, resource string) {
```

Comment:
Some comments and a more bit more elaborate name please

Reply:
Added comments but GVR/GVK and CR are widely used terms in operator hub so I've kept the function name as-is. If you'd anyway like me to make it `getGroupVersionResourceFromCustomResource`, let me know. I'll do it.

Reply:
OK let's keep it short :+1: